### PR TITLE
New package: x86info

### PR DESCRIPTION
**Pull request by foxlet**

### CODE REVIEW

File: srcpkgs/x86info/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'x86info'
+pkgname=x86info
+version=1.30
+revision=1
+only_for_archs="x86_64 x86_64-musl i686 i686-musl"
```
Comment (Cogitri) [inline srcpkgs/x86info/template]: Wouldn't `nocross` have the same affect as his?

Comment (Johnnynator) [reply srcpkgs/x86info/template]: Nope. nocross would still allow to build it natively on e.g. arm*. This package should be only ever build able for a x86 target arch. But arm to x86 cross compilation theoretically could be allowed.

Comment (Cogitri) [reply srcpkgs/x86info/template]: Ah, right, only thought about how the builders build it here, thanks for correcting me here :)

File: srcpkgs/x86info/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+distfiles="http://codemonkey.org.uk/projects/x86info/x86info-${version}.tgz"
+checksum=fe3009850e93a9c5fa38ed6bf74d72686043329fead8f1b2be4b5943d4fc9428
+
+case "$XBPS_TARGET_MACHINE" in
+	*-musl) CFLAGS="-D_GNU_SOURCE";;
+esac
+
+do_install() {
+	vinstall x86info 775 usr/bin/x86info
+	vinstall lsmsr 775 usr/bin/lsmsr
```
Comment (cr6git) [inline srcpkgs/x86info/template]:
````
vinstall x86info 775 usr/bin
vinstall lsmsr 775 usr/bin
````

Comment (Johnnynator) [reply srcpkgs/x86info/template]: wouldn't using `vbin x86info` and `vbin lsmsr` be nicer?

Comment (cr6git) [reply srcpkgs/x86info/template]: `vbin` installs file into usr/bin with permissions 755.

Comment (cr6git) [reply srcpkgs/x86info/template]: @foxlet perhaps you could explain why you want 775 instead of 755 ?

File: srcpkgs/x86info/template [hunk 3]
```diff
@@ -18,8 +18,8 @@ case "$XBPS_TARGET_MACHINE" in
 esac
 
 do_install() {
-	vinstall x86info 775 usr/bin/x86info
-	vinstall lsmsr 775 usr/bin/lsmsr
+	vinstall x86info
+	vinstall lsmsr
```
Comment (Cogitri) [inline srcpkgs/x86info/template]: I think you mean to do `vbin` here

**Comment by Cogitri** [comment]
Also, squash those commits into one

**Comment by foxlet** [comment]
No need to say, this filesystem is case-insensitive, so it was having issues squashing.

**Pull request merged**